src/components/Navbar.tsx

```diff
@@ -0,0 +1,19 @@
+import { FC, memo } from 'react';
+import { TemplateNavLink } from './TemplateNavLink';
+
+export const Navbar: FC = memo(() => {
+  return (
+    <nav
+      className="navbar is-light is-fixed-top is-mobile
+      has-shadow has-navbar-fixed-top"
+      data-cy="Nav"
```

Comment:
indentations deserve to be better here
```suggestion
      className="navbar is-light is-fixed-top is-mobile
      has-shadow has-navbar-fixed-top"
      data-cy="Nav"
```